ajax/patterns-01-en.json

```diff
@@ -1,5 +1,6 @@
 ---

```

Comment:
Should the JSON file actually include this frontmatter? It's red because the parser is flagging that it's invalid

Reply:
Yes because some URLs are pulling a variable from the config.yml file. We will change sunset this file for a more native mechanism, this year.